src/chrome/content/rules/GolfLink.com.xml

```diff
@@ -1,34 +1,13 @@
 <!--
-	For other Demand Media coverage, see Demand-Media.xml.
-
-
-	CDN buckets:
-
-		- i.glimg.net.edgesuite.net
-
-			- a1768.g.akamai.net
-
-		- ui.glimg.net.edgesuite.net
-
-			- a1075.g.akamai.net
+	For rules that are on by default, see Demand-Media.xml.
 
+	Non-functional hosts
+		Timeout was reached:
+		- blog.golflink.com
 -->
 <ruleset name="GolfLink.com (partial)">
-
-	<target host="i.glimg.net" />
-	<target host="ui.glimg.net" />
 	<target host="golflink.com" />
-	<target host="cdn-www.golflink.com" />
-	<target host="golflink.net" />
-	<target host="www.golflink.net" />
```

Comment:
Why did you remove all other targets?

Reply:
I missed some targets when I was trying to extract them from the ruleset. I pushed more changes to includes more targets. Thanks.